axlearn/common/trainer.py

```diff
@@ -150,6 +150,9 @@ class Config(Module.Config):
         # An optional recorder for measuring common metrics like step time.
         recorder: Optional[InstantiableConfig[measurement.Recorder]] = None
 
+        # The frequency of logging during training.
+        log_every_n_steps: Optional[int] = None
```

Comment:
did not set `log_every_n_steps: int = 100` to avoid a large amount of config.txt update. If we think it makes more sense for the change, we can also change it